src/test/java/fi/otavanopisto/kuntaapi/test/utils/PtvDownload.java

```diff
@@ -0,0 +1,243 @@
+package fi.otavanopisto.kuntaapi.test.utils;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import ezvcard.util.IOUtils;
+import fi.metatavu.ptv.client.model.V5VmOpenApiOrganizationService;
+import fi.metatavu.ptv.client.model.V7VmOpenApiOrganization;
+import fi.metatavu.ptv.client.model.V7VmOpenApiService;
+import fi.metatavu.ptv.client.model.V7VmOpenApiServiceServiceChannel;
+import fi.otavanopisto.kuntaapi.server.integrations.ptv.servicechannels.ServiceChannelType;
+
+@SuppressWarnings ({"squid:S106", "squid:S3457", "squid:S1148"})
+public class PtvDownload {
+  
+  private static final String JSON = ".json";
+  private static final String PTV = "v7";
+
+  public static void main(String[] args) throws IOException {
+    new PtvDownload()
+      .downloadCodes();
+  }
+  
+  private PtvDownload downloadCodes() throws MalformedURLException, FileNotFoundException, IOException {
+    String[] areaCodeTypes = {"Province", "HospitalRegions", "BusinessRegions"};
+    for (String areaCodeType : areaCodeTypes) {
+      download(getFullUrl(String.format("/CodeList/GetAreaCodes/type/%s", areaCodeType)), new File(getCodesFolder(), String.format("%s.json", areaCodeType.toLowerCase())));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "%s.json" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)